packages/react-components/src/components/InputBoxComponent.test.tsx

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { InputBoxComponent } from './InputBoxComponent';
+import { render, screen, waitFor, act, fireEvent } from '@testing-library/react';
+// /* @conditional-compile-remove(mention) */
+import userEvent from '@testing-library/user-event';
+// /* @conditional-compile-remove(mention) */
+import { Mention } from './MentionPopover';
+
+describe('InputBoxComponent should call onChange', () => {
+  test('onChange callback should return correct value', async () => {
+    let changedValue = '';
+    render(
+      <InputBoxComponent
+        inlineChildren={true}
+        textValue={changedValue}
+        placeholderText="Enter a message"
+        onChange={(_, newValue) => {
+          changedValue = newValue ?? '';
+        }}
+        maxLength={1000}
+      >
+        <div></div>
```

Comment:
I'm curious as to why the div is needed here?

Reply:
`InputBoxComponent` has a required children props and can't be used as a self closing tag